xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1165,6 +1169,25 @@ bool CDVDDemuxFFmpeg::SeekTime(double time, bool backwards, double* startpts)
 
   if (m_checkvideo)
   {
+    XbmcThreads::EndTime timer(1000);
+
+    while (!IsVideoReady())
+    {
+      DemuxPacket* pkt = Read();
+      if (pkt)
+        CDVDDemuxUtils::FreeDemuxPacket(pkt);
+      else
+        Sleep(10);
+      m_pkt.result = -1;
+      av_packet_unref(&m_pkt.pkt);
+
+      if (timer.IsTimePast())
+      {
+        CLog::Log(LOGERROR, "%s - Timed out waiting for video to be ready", __FUNCTION__);
```

Comment:
I never liked __FUNCTION__ because it does not print the class name on Linux.

Reply:
I saw that half the log statements used it and the other half didn’t.

Reply:
No issue with changing it if that’s your preference

Reply:
I don't care much about preferences in this project anymore. I just let you know what I learned in the past. Consider a log line "SeekTime - ...". Finding the correct SeekTime method may not be fun:
https://github.com/xbmc/xbmc/search?q=SeekTime&unscoped_q=SeekTime

Reply:
Completely fair. Done.